util/binfmt_misc/generate.go

```diff
@@ -0,0 +1,91 @@
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"flag"
+	"fmt"
+	"html/template"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+// saves baseimage binaries statically into go code
+func main() {
+	flag.Parse()
+	if len(flag.Args()) == 0 {
+		panic(fmt.Errorf("arch is required"))
+	}
+
+	for _, arch := range flag.Args() {
+		wd, err := os.Getwd()
+		if err != nil {
+			panic(err)
+		}
+		f, err := os.Open(filepath.Join(wd, arch))
+		if err != nil {
+			panic(err)
+		}
+		defer f.Close()
+		buf := &bytes.Buffer{}
+
+		gz, err := gzip.NewWriterLevel(newHexStringWriter(buf), gzip.BestCompression)
```

Comment:
is it worth gzipping?

Reply:
It saves a little bit in binary size. They seem to compress quite well.